doc/content/sensor-edition/beta-welcome/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "WELCOME, WE ARE EXCITED!"
+description: ""
+weight: -1
+distributions: null
+---
+
+If you are reading this, you are probably one of the first people to get their hands on the {{% gnse %}}.
+We are extremely excited to have you as part of the community.
+
+<!--more-->
+
+The {{% gnse %}} is a unique device for multiple reasons:
+- It is a functional device that can be used for many LoRaWAN applications
+- It is a production ready device that incorporates ST new STM32WL SOC
+- It is a reference design hardware
+- It has a source available firmware
+
+## A developers Beta rollout
+
+The {{% gnse %}} is currently in Beta phase.
+
+This means a number of things:
+1. You are one of the few people who can actually experience the device hardware and firmware
+2. You can actually contribute to the development of the device
+
+So, as part of this Beta phase we expect you to:
+- Be able to flash, debug and interact with the device
+- Be able to connect external serial loggers, debuggers and sensors
+- Contribute to the development of the device by raising issues
+- Develope and tryout different applications
+
+And you can expect us to:
+- Prioritize any issue that you face
+- Your feedback will be analyzed and deeply considered
+
+## Where to start?
+
+The [Getting started]({{< ref "/getting-started" >}}) is a great place to get familiar with the device.
+
+## Where to contribute?
+
+We use the <a href="https://github.com/TheThingsIndustries/generic-node-se" target="_blank">{{% gnse %}} Github repository</a> in our development. You can <a href="https://github.com/TheThingsIndustries/generic-node-se/issues/new/choose" target="_blank">file an issue</a> to request a feature or report a bug.
```

Comment:
We typically use markdown syntax for urls, so

```
[Github repository](https://github.com/TheThingsIndustries/generic-node-se)
```

For internal links, use the `ref` shortcode -> https://gohugo.io/content-management/shortcodes/#ref-and-relref

Reply:
Interesting, I will update this later